docs/community.rst

```diff
@@ -0,0 +1,29 @@
+.. _community:
+
+Community
+=========
+
+We welcome your feedback, questions and ideas.
+Here's how to reach the community.
+
+Code of Conduct
+---------------
+
+All communication and interactions in the Ansible community are governed by the `Ansible code of conduct <https://docs.ansible.com/ansible/devel/community/code_of_conduct.html>`_.
+Please read and abide by it!
+Reach out to our community team at `codeofconduct@ansible.com <mailto:codeofconduct@ansible.com>`_ if you have any questions or need assistance.
+
+Ansible Forum
+-------------
+
+Join the `Ansible Forum <https://forum.ansible.com>`_, the default communication platform for questions and help, development discussions, events, and much more.
+`Register <https://forum.ansible.com/signup?>`_ to join the community.
+Search by categories and tags to find interesting topics or start a new one; subscribe only to topics you need!
+
+* `Get Help <https://forum.ansible.com/c/help/6>`_: get help or help others. Please add appropriate tags if you start new discussions, for example `ansible-runner`, `playbook`, and  `awx`.
```

Comment:
This line is generating warnings in the docs build, causing the build failures. Perhaps you meant to use `` instead of `.

Reply:
@Shrews Thanks. Yeah I think that's it. I was switching between markdown and rst and got caught out. I'll fix it now. Sorry I didn't catch it myself.